packages/README.md

```diff
@@ -9,9 +9,9 @@ Currently, supported commands include package action commands and statistical co
 
 ###Package action commands
 
-* **load** package_name [package_name] ...
+* **load** package_name sha1_checksum [package_name sha1_checksum] ...
```

Comment:
Should we also document or provide a utility to the end-user to calculate checksum. Thus, if he writes a package, he can run the utility to calculate checksum using our utility method and then use that checksum to load the package.